### DISCUSSION TITLE
CVC5 webinterface lang field and SyGuS.

### DISCUSSION THREAD

**Discussion by jparsert** [cvc5/cvc5#9809] [General]
In the cvc4 web-interface there used to be a drop-down menu on the right to chose sygus input format. This seems to be gone in cvc5 and I don't know how to input SyGuS problems in the web interface (https://cvc5.github.io/app/). Adding sygus or --sygus to the arguments field does not seem to work. Also, there seems to be a "lang" field in the top right but the drop down menu seems to be gone. Some quick inspecting of the HTML code indicates that there should be some key/value store there. Not sure if there is something missing.

**Comment by mudathirmahgoub** [comment]
Hi @jparsert 
Thanks for reporting this issue. 
It should be fixed after this PR is merged. 
https://github.com/cvc5/cvc5.github.io/pull/8
